src/wp-includes/blocks.php

```diff
@@ -670,9 +670,11 @@ function has_blocks( $post = null ) {
  *
  * This test optimizes for performance rather than strict accuracy, detecting
  * whether the block type exists but not validating its structure and not checking
- * reusable blocks. For strict accuracy, you should use the block parser on post content.
+ * patterns (previously referred to as reusable blocks). For strict accuracy, you
```

Comment:
I'm concerned term "patterns" may be confusing. Looking at the documentation and UI callouts for the renaming, they are being referred to as "synced pattern" https://wordpress.org/documentation/article/reusable-blocks/.